File: API/Models/FolhaDePagamento.cs
using System;

namespace API.Models
{
    public class FolhaDePagamento
    {
        public int Id { get; set; }
        public int Mes { get; set; }
        public int Ano { get; set; }
        public int Horas { get; set; }
        public int ValorHora { get; set; }
        public Funcionario Funcionario { get; set; }
        public int FuncionarioId { get; set; }
        public DateTime CriadoEm { get; set; }

       public FolhaDePagamento() => CriadoEm = DateTime.Now;

    }
}

File: FRONT/src/app/models/folhaDePagamento.ts
import { Funcionario } from "./funcionario";

export interface FolhaDePagamento {
    folhaId?: number;
    mes: number;
    ano: number;
    horas: number;
    valorHora: number;
    funcionario?: Funcionario;
    funcionarioId: number;
    CriadoEm?: Date;
}
